programs/drift/src/instructions/keeper.rs

```diff
@@ -205,18 +205,26 @@ fn fill_spot_order<'a, 'b, 'c, 'info>(
 
     let (_referrer, _referrer_stats) = get_referrer_and_referrer_stats(remaining_accounts_iter)?;
 
-    let mut fulfillment_params = match fulfillment_type {
+    let mut fulfillment_params: Box<dyn SpotFulfillmentParams> = match fulfillment_type {
         Some(SpotFulfillmentType::SerumV3) => {
             let base_market = spot_market_map.get_ref(&market_index)?;
             let quote_market = spot_market_map.get_quote_spot_market()?;
-            get_serum_fulfillment_accounts(
+            Box::new(get_serum_fulfillment_params(
```

Comment:
This is another nitpick, but I think this would be easier to understand if this were a static method.
```
SerumFulfillmentParams::new(...)
```

Reply:
yeah we could do this, was also thinking we could maybe make a method on the SpotFulfillmentType to get the corresponding params struct